solidity/contracts/OracleAggregator.sol

```diff
@@ -15,7 +16,7 @@ contract OracleAggregator is AccessControl, SimpleOracle, IOracleAggregator {
   mapping(bytes32 => OracleAssignment) internal _assignedOracle; // key(tokenA, tokenB) => oracle
 
   constructor(
-    ITokenPriceOracle[] memory _initialOracles,
+    address[] memory _initialOracles,
```

Comment:
Conceptually, until we check if they are oracles, they are just addresses

If felt weird to ask an oracle if they were an oracle haha

Since casting doesn't have any gas costs, I thought this made more sense

Reply:
Yeah! It's the right approach! Gr8 work!